File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 3- Using the ternary operator/PositiveAndNegativeNumber.cpp
#include<iostream>
using namespace std;

int main()
{
    int num;
    //take input
    cout<< "Enter the number:";
    cin>>num;

    //Condition to check if the 0, positive or negative
    if(num == 0)
        cout<< "Zero";
    else (num > 0)? cout <<"Positive": cout<<"Negative";

    return 0;
}

File: Top 100 Codes/001-check if a Number Is Positive Or Negative/Method 3- Using the ternary operator/Positive_And_Negative_Number.java
import java.util.Scanner;

public class Positive_And_Negative_Number {
    public static void main(String[] args) {
        //using Ternary Operator
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter a Number: ");
        int num =sc.nextInt();

        //Condition to check if the number is negative or positive

        if (num ==0)
        {
            System.out.println("Zero");
        }
        else{
            String result = num > 0 ?"The number is Positive":"The number is negative";
            System.out.println(result);
        }
        sc.close();
    }
    
}
